embl_robot.py

```diff
@@ -207,7 +207,7 @@ def preMount(self, gov_robot, puckPos, pinPos, sampID, **kwargs):
             prefix90 = sampName + "_" + str(puckPos) + "_" + str(pinPos) + "_" + str(reqCount) + "_PA_90"
             kwargs['prefix1'] = prefix1
             kwargs['prefix90'] = prefix90
-            top_view.topViewSnap(prefix1,getBlConfig("visitDirectory")+"/pinAlign",1,acquire=0)
+            # top_view.topViewSnap(prefix1,getBlConfig("visitDirectory")+"/pinAlign",1,acquire=0)
```

Comment:
any reason this can't just be removed?

Reply:
Currently keeping this just for FMX (code now has a beamline==FMX check) until their top cameras are in place until then the old top_view can do the initial alignment